frontend/components/vault/Vault.vue

```diff
@@ -0,0 +1,278 @@
+                                    </template>
+                                    <span v-else class="opacity-50">Unknown</span>
+                                </td>
+                            </tr>
+                            <tr class="bg-gray-100 dark:bg-gray-800">
+                                <td>Vault debt</td>
+                                <td>
+                                    <FormatCurrency
+                                        v-if="vaultTransaction.initialDebtDai"
+                                        :value="vaultTransaction.initialDebtDai"
+                                        currency="DAI"
+                                    />
+                                    <span v-else class="opacity-50">Unknown</span>
+                                </td>
+                            </tr>
+                            <tr class="bg-gray-100 dark:bg-gray-800">
+                                <td>Vault address</td>
+                                <td>
+                                    <format-address
+                                        v-if="vaultTransaction.address"
+                                        :value="vaultTransaction.address"
+                                        disable
+                                    />
+                                    <span v-else class="opacity-50">Unknown</span>
+                                </td>
+                            </tr>
+                        </template>
+                    </tbody>
+                </table>
+                <div v-if="error" class="Disable" />
+            </div>
+            <button
+                class="
+                    w-full
+                    border-2 border-gray-300
+                    p-1
+                    border-t-0
+                    text-center text-gray-400
+                    dark:border-gray-600 dark:text-gray-600
+                "
+                @click="toggleExpandable"
+            >
+                {{ isTableExpanded ? 'Hide additional info' : 'Show additional info' }}
+            </button>
+            <template v-if="isExplanationsShown">
+                <TextBlock class="mt-4">
+                    <template v-if="!error">
+                        The vault
+                        <format-address type="address" :value="vaultTransaction.address" shorten disable /> contains
+                        <format-currency
+                            :value="vaultTransaction.collateralAmount"
+                            :currency="vaultTransaction.collateralType"
+                        />. The initial debt taken by the vault is
+                        <format-currency :value="vaultTransaction.initialDebtDai" currency="DAI" />. The current
+                        proximity to liquidation is
+                        <AnimatedNumber :value="vaultTransaction.proximityToLiquidation" />. The next price update will
```

Comment:
How many symbols will `AnimatedNumber` display after the floating point by default or all of them?

Reply:
> by default 

By default. In other words, if `0.123456789` is provided what will be displayed?

Reply:
It displays the first number after the decimal point, that is not zero and the preceding number. Therefore:

`0.12`